src/cpp/library/new

```diff
@@ -21,33 +21,36 @@ struct nothrow_t {
 extern const nothrow_t nothrow;
 
 namespace std {
+/**
+	*source: https://en.cppreference.com/w/cpp/memory/new/bad_alloc
```

Comment:
white space indent

Reply:
@XLiZHI : Please address @kunjsong01's comment before we merge this PR.

Reply:
done